apps/web/pages/new-booker/team/[slug]/[type].tsx

```diff
@@ -30,6 +30,7 @@ export default function Type({ slug, user, booking, away, isBrandingHidden }: Pa
         rescheduleBooking={booking}
         isAway={away}
         hideBranding={isBrandingHidden}
+        isTeamEvent
```

Comment:
New boolean in booker that can now force it to only fetch team events.

Reply:
We would probably have to do it for private links as well. A private link can be a team event or user event.